audius-cli

```diff
@@ -425,6 +429,7 @@ def launch(ctx, service, seed, chain, yes):
 
     if service == "discovery-provider":
         ctx.invoke(launch_chain)
+        ctx.invoke(launch_required_plugins)
```

Comment:
not really related to this diff, but should notifications be considered part of required plugins if its not already? i guess it's not technically required for the app to run.. lmk what you think

Reply:
Its not required because we only run it on one dn, relay should be on each one

Reply:
oh right makes sense!